scripts/python/services/test_GithubService.py

```diff
@@ -302,5 +303,57 @@ def test_throws_exception_when_client_throws_exception(self, mock_github_client_
             ConnectionError, github_service.create_new_team_with_repository, "test_team", "test_repository")
 
 
+@patch("gql.transport.aiohttp.AIOHTTPTransport.__new__", new=MagicMock)
+@patch("gql.Client.__new__", new=MagicMock)
+@patch("github.Github.__new__")
+class TestGithubServiceTeamExists(unittest.TestCase):
+    mock_unicorn_team = None
+    mock_super_team = None
+
+    def setUp(self):
+        mock_unicorn_team, mock_super_team = Mock(Team), Mock(Team)
+        mock_unicorn_team.name = "unicorn,team"  # `name` is a reserved value in `Mock()` constructors. So need to mock the values manually.
```

Comment:
Took me a while to realise that `name` was a reserved value in the `Mock()` constructor. Left a comment to highlight this issue for future reference 👀 	

[REF](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock)